src/Components/Navbar.js

```diff
@@ -5,15 +5,22 @@ import TabContext from '@material-ui/lab/TabContext';
 import TabList from '@material-ui/lab/TabList';
 import styled from 'styled-components';
 import TabPanel from "@material-ui/lab/TabPanel";
-import FloatButton from "./Floating-Action-Bottun";
+import Fab from "@material-ui/core/Fab";
+import AddIcon from '@material-ui/icons/Add';
 
 const AppBarStyled = styled(AppBar)`
         background : #222;
     `;
 
 const TabContextStyled = styled(TabContext)`
         flex-grow: 1;
-`
+`;
+
+const FabStyled = styled(Fab)`
+    position: absolute;
+    right: 16px;
+    bottom: 16px;
+`;
 
 export default function NavBar() {
```

Comment:
In navigation bar component, `NavBar`, let's do a better naming for the variable that keeps the state for the navigation, I suggest:
```
const [tabIndex, setTabIndex] = React.useState(String(1));
```
This explains better what is the purpose for the state variable. Also It's better to convert a number to a string using the. `String` function. Because I think it's more declarative. So make the subsequent changes for this to work.